packages/office-ui-fabric-react/src/components/Keytip/Keytip.styles.ts

```diff
@@ -0,0 +1,77 @@
+import { IKeytipStyleProps, IKeytipStyles } from './Keytip.types';
+import { ICalloutContentStyleProps, ICalloutContentStyles } from '../../Callout';
+import { IStyleFunction, IPoint } from '../../Utilities';
+
+export const getStyles = (props: IKeytipStyleProps): IKeytipStyles => {
+  const { theme, disabled, visible } = props;
+  return {
+    container: [
+      {
+        backgroundColor: theme.palette.neutralDark
+      },
+      disabled && {
+        opacity: 0.5,
+      },
+      !visible && {
+        visibility: 'hidden',
+        opacity: 0
```

Comment:
> opacity: 0 [](start = 7, length = 11)

is setting opacity here needed if we are setting visibility to hidden?

Reply:
Opacity: 0 shouldn't matter if we are setting visibility: hidden (or display: none), we need to make sure this reads correctly in IE. There were issues in the past (in some cases) where setting visibility: hidden and/or display: none would make it to where the screen reader would not read out the contents with that styling applied. Setting opacity: 0 was the workaround to have the content not be visible but still allow the screen reader to see it. We should double check that things read out correctly with visibility: hidden set